javascript.js

```diff
@@ -0,0 +1,20 @@
+// set variavles
+var user = []; //inline coment
+
+/**
+ * Adds two numbers togeter
+ * 
+ * @param {number} first the first number
+ * @returns {number}
+ */
+function add(first) {
+  /*
```

Comment:
Typo inside comment in token 'coment'. Possible corrections:comment (59),moment (40),cement (38)